controllers/activemqartemis_controller_test.go

```diff
@@ -682,6 +683,69 @@ var _ = Describe("artemis controller", func() {
 		})
 	})
 
+	Context("PVC upgrade owner reference test", Label("pvc-owner-reference-upgrade"), func() {
+		It("faking a broker deployment with owned pvc", func() {
+			crd := generateArtemisSpec(defaultNamespace)
+			crd.Spec.DeploymentPlan.Size = 1
+			crd.Spec.DeploymentPlan.PersistenceEnabled = true
+
+			if os.Getenv("USE_EXISTING_CLUSTER") == "true" {
```

Comment:
@gaohoward I can't recall, does this log when this condition is not met and the bulk of the test is actually not run? Does it show as a skip, or a pass? If the latter making a note via a logged message in the else case might be useful although obviously not necessary.

Reply:
+1, the USE_EXISTING_CLUSTER means it only run if an existing cluster (like minikube). Otherwise it just sliently skip the test. (see test-mk target). I'll add a log message.